### PR TITLE
Shorten given list of extensions

### PR DESCRIPTION
**Pull request by kevinhaube**
Story: https://qmacbis.atlassian.net/browse/OY2-24878

### Details

Adjusts the copy for attachment file types text.

### Changes

- N/A

### Implementation Notes

- N/A

### Test Plan

1. Navigate to the dashboard as a submitting user
2. Use "New Submission" button and triage options to navigate to each form page
3. Ensure each form page shows the proper copy for attachment file types

### CODE REVIEW

**Comment by kevinhaube** [comment]
https://dr7z9fwgoblun.cloudfront.net/

**Review by kristin-at-theta — APPROVED** [review]
Changes look great and follow the AC for the file list.

I clicky-clicky'd through all the forms and they all show the same list (no weird overwriting somewhere) and the FAQ link goes to the correct section.

Approved! -- and then I re-read the AC!  oops :)

File: services/ui-src/src/libs/formLib.tsx [hunk 1]
```diff
@@ -54,11 +54,7 @@ type ConfirmSubmitType = {
 
 export const DefaultFileTypesInfo = () => (
   <p>
-    We accept the following file types:{" "}
-    <b>
-      .doc, .docx, .jpg, .odp, .ods, .odt, .png, .pdf, .ppt, .pptx, .rtf, .txt,
-      .xls, .xlsx, and a few others.
-    </b>{" "}
+    We accept the following file types: <b>.docx, .jpg, .pdf, .png, .xlsx.</b>{" "}
```
Comment (kristin-at-theta) [inline services/ui-src/src/libs/formLib.tsx:57]: Though WHOOOPS - missed that the AC says file "formats" not "types" -- probably to keep from being confused with attachment types?!

**Review by kristin-at-theta — CHANGES_REQUESTED** [review]
file types should be file formats per the AC

**Review by kristin-at-theta — APPROVED** [review]
Hooray!

Approved (again)

**Pull request merged**